gen_updates_cnn/README.md

```diff
@@ -0,0 +1,78 @@
+
+# Data-Parallel Distributed Supervised Learning with libEnsemble
+
+`python run_libe_cnn.py -n N`
+
+The training dataset is evenly split among the N workers. libEnsemble
+starts N parallel training instances on separate, distributed
+worker processes. These workers compute the gradient of the loss function for their portion
+of the dataset and send their gradients to a generator process. The generator process
+combines the gradients from all workers, updates the model parameters, and sends 
+the updated model back to the workers.
+
+Given the number of gradients/parameters that are being updated, `proxystore`
+is used.
+
+## Justification
+
+If the local dataset for training a single model must remain small,
+then time-to-generalization during training may be high as batches must be loaded
+and unloaded. Data-parallel or cross-node techniques in many AI libraries are difficult
+to run on multi-node systems, or assume user familiarity with parallel programming.
+
+libEnsemble allows easy parallelization across multiple nodes and HPC
+systems. "Ensembles" of experiments with libEnsemble take no configuration
+to run on separate nodes and intercommunicate. 
+
+In this example, one model is optimized in parallel by a parent "generator"
```

Comment:
I don't quite get this sentence ‘optimized in parallel by a parent "generator"’. It's been done by the generator than that part is not in parallel.